sigstore-gradle/sigstore-gradle-sign-base-plugin/src/main/kotlin/dev/sigstore/sign/tasks/SigstoreSignFilesTask.kt

```diff
@@ -159,10 +175,28 @@ abstract class SigstoreSignFilesTask : DefaultTask() {
     protected fun sign() {
         // Ensure task holds a lock, so no other signign tasks executes concurrently
```

Comment:
I think this comment maybe is unncessary now.

Reply:
It is a comment to signingservice.get
Serialization is needed so Gradle does not spawn multiple worker processes as each process would perform its own OIDC dance